### PR TITLE
Changing Registration Flow

### PR DESCRIPTION
**Pull request by mike-zaharchenko17**
Changes:
- Edited registration flow in views.py; now, when a user registers (creates an account), they will be automatically logged in and redirected to the edit_profile view, where they will be encouraged to fill out their profile information.
- Defined a helper function, return_lang_tuple, and set its callable to be the default value for the languages field in UserProfile model. This will set English as a default language for a user.
- Added max and min value validators for age_lower and age_upper. They will make sure that these fields only accept inputs between 18 and 99 as the age range.
- Changed default value for age_lower to 18.

### CODE REVIEW

File: user_profile/helpers.py [hunk 1]
```diff
@@ -35,3 +35,7 @@ def dob_gte18_and_lt100(dob):
                 False,
                 "The age you entered is over 99. Please verify your inputs and try again",
             )
+
+
+def return_lang_tuple():
```
Comment (kravisankaran) [inline user_profile/helpers.py:40]: Can we rename this to get_default_lang_tuple?

File: user_profile/models.py [hunk 2]
```diff
@@ -25,14 +27,16 @@ class UserProfile(models.Model):
...
+        default=18,
         verbose_name="Age(lower bound)",
         help_text="Enter lower bound for age filter",
+        validators=[MinValueValidator(18), MaxValueValidator(99)],
     )
     age_upper = models.IntegerField(
         default=99,
         verbose_name="Age(upper bound)",
         help_text="Enter upper bound for age filter",
+        validators=[MinValueValidator(18), MaxValueValidator(99)],
```
Comment (kravisankaran) [inline user_profile/models.py:39]: Does min have a hard check on 18, is it only an integer check as opposed to a decimal?

**Review by kravisankaran — CHANGES_REQUESTED** [review]
Left a few comments

File: user_profile/helpers.py (hunk 1 above)
Comment (mike-zaharchenko17) [reply user_profile/helpers.py:40]: Sure

**Review by kravisankaran — APPROVED** [review]

**Pull request closed without merging**